proposals/memory-control/Overview.md

```diff
@@ -81,23 +124,70 @@ WebAssembly.instantiateStreaming(fetch('memory.wasm'), {js: { memory }})
 
 ##### TODO: memory.discard & memory.protect APIs
 
+
+## Suggested Implementation
+
+The general expectation is that creating a mappable memory will act as a "reserve", allocating address space in the process but not assigning any physical resources. On Windows, this would be a `VirtualAlloc(MEM_RESERVE)`; on macOS and Linux this would be `mmap(MAP_FIXED, PROT_NONE)` (which works even on systems without overcommit).
+
+### Core instructions
+
+The core `memory.map` would act as a "commit", (lazily) assigning physical resources to the pages. On Windows this would be a `VirtualAlloc(MEM_COMMIT, <prot>)`; on macOS and Linux this would be `mmap(MAP_FIXED, <prot>)`.
```

Comment:
Forgot to add in the previous review: I wonder how much detail is worth adding here, I can imagine making recommendations about the initial memory allocation here could also be useful.